### PR TITLE
Merge 'view user stats' menu functionality and other small changes into main oop branch

### PR DESCRIPTION
**Pull request by robynfsj**
Have added the option to view the user stats. But also made a few change to other parts. Just checking you are ok for me to merge these in case my new changes break anything you might have done on the register user vehicle functions.

### CODE REVIEW

File: cli_components.py [hunk 1]
```diff
@@ -24,14 +23,26 @@ def welcome_banner():
               f"\n\n(To exit enter 'q')")
 
     @staticmethod
-    def user_main_menu():
+    def display_main_menu():
```
Comment (robynfsj) [inline cli_components.py:26]: simple rename so more self explanatory

File: cli_components.py [hunk 2]
```diff
@@ -24,14 +23,26 @@ def welcome_banner():
...
         """Prints the main user menu."""
         print("Please select from the following options:\n\n"
               "    (1) plan journey\n"
               "    (2) view user stats\n"
               "    (3) register a vehicle\n"
               "    (4) log out\n")
 
+    @staticmethod
```
Comment (robynfsj) [inline cli_components.py:34]: Function that prints out user stats. Simple for now but we can make better later.

File: main.py [hunk 3]
```diff
@@ -136,9 +139,10 @@ def reg_message():
             Db.new_user(self.fname, self.lname, self.email, pword)
             self.user_id = Db.get_new_user_id(self.email)
             self.logged_in = True
+            self.main_menu()
 
 
-    def user_action(self):
+    def main_menu(self):
```
Comment (robynfsj) [inline main.py:145]: Again, simple name change but much more understandable

File: main.py [hunk 4]
```diff
@@ -150,39 +154,40 @@ def user_action(self):
...
-            elif option == "2":
-                print("display stats function is called here")
-                # self.display_stats()
-            elif option == "3":
-                print("register vehicle function is called here")
-                # self.register_vehicle()
-            elif option == "4":
-                self.logout()
-            else:
+            try:
```
Comment (robynfsj) [inline main.py:165]: Probably the biggest change. Have just moved it all into try except statement.

**Comment by Laksh-13** [comment]
> Have added the option to view the user stats. But also made a few change to other parts. Just checking you are ok for me to merge these in case my new changes break anything you might have done on the register user vehicle functions.

Feel free to close this branch and merge in the OOP one..:)

**Review by Laksh-13 — APPROVED** [review]

**Pull request merged**